headless-browser/src/browser.js

```diff
@@ -0,0 +1,14 @@
+import which from 'which';
+import { chromium } from 'playwright-core';
+
+/**
+ * @param {number} port
+ */
+export async function startBrowser(port) {
+  const browser = await chromium.launch({
+    executablePath: await which('chromium'),
```

Comment:
I think there some problem, If somebody use MacOS Or Windows, Is not working

Maybe should `const browser = await chromium.launch()`

Before install (cue in document)
```bash
npx playwright install
```